vocabularies/Common.xml

```diff
@@ -47,48 +48,48 @@
       </Term>
 
       <!-- Section: Versioning -->
-      <Term Name="ServiceVersion" Type="Edm.Int32" AppliesTo="Schema">
+      <Term Name="ServiceVersion" Type="Edm.Int32" Nullable="false" AppliesTo="Schema">
         <Annotation Term="Core.Description" String="1 for first version of a service, incremented when schema changes incompatibly and service is published with a different URI" />
       </Term>
-      <Term Name="ServiceSchemaVersion" Type="Edm.Int32" AppliesTo="Schema">
+      <Term Name="ServiceSchemaVersion" Type="Edm.Int32" Nullable="false" AppliesTo="Schema">
         <Annotation Term="Core.Description" String="0 for first schema version within a service version, incremented when schema changes compatibly" />
       </Term>
 
       <!-- Section: General Semantics -->
-      <Term Name="Label" Type="Edm.String">
+      <Term Name="Label" Type="Edm.String" Nullable="false">
```

Comment:
Terms containing a human-readable text should remain nullable, because they might be annotated using an `Edm.Path`, which may point to `null`.

Reply:
@BerSie How would the UI deal with a label that is null?

Reply:
With an empty label (I would expect and hope for it)

Reply:
@BerSie so we can set it explicitly to nullable=true?

Reply:
But `Common.Text` is typically a property, this must definitively remain nullable.

Reply:
I would like to test it before saying anything wrong ... :-)

Reply:
> But `Common.Text` is typically a property, this must definitively remain nullable.

CHAR/STRING properties in ABAP are never nullable, so in that prominent use case Common.Text never is null.

Reply:
Unfortunately it's not handled like I expected (don't ask me what the result is ... it left me speechless). I personally would stay for Common.Label with Nullable="false". For Common.Text I'm not sure if we can/should prevent the referenced property to be null (for non-ABAP platforms).

Reply:
Done.